src/schema/schema-generators.ts

```diff
@@ -56,11 +55,8 @@ export function rawProp(name: string): [RawBlockPropDefinition] {
 	return [{ name, raw: true }];
 }
 
-export function inlineProp(
-	name: string,
-	...content: Nested<InlinePropContentDefinition>
```

Comment:
If we're getting rid of `Nested<T>` here, we should get rid of it everywhere.

The original idea of this pattern of `...x: Nested<T>` and `x.flat()` was that you should be able to mix lists and single args without spreading:

```typescript
const inlines = ["strong", "link"].map(ref)
const schema = {
    ...
        head: inlineProp("name", "contentOne", inlines, "contentTwo")
    ...
}
```

But perhaps this is overly complicating things for no good reason.  Instead, we'd have to do this without `Nested`:

```typescript
inlineProp("name", "contentOne", ...inlines, "contentTwo")
```

So I vote that we get rid of this `Nested<T>` thing altogether.

Reply:
Done ✅